kong/plugins/rate-limiting/schema.lua

```diff
@@ -19,6 +19,7 @@ return {
     redis_timeout = { type = "number", default = 2000 },
     redis_database = { type = "number", default = 0 },
     hide_client_headers = { type = "boolean", default = false },
+    http_header = { type = "string" }
```

Comment:
pedantic style: please append a comma to final table elements to reduce future diffs.